### ISSUE TITLE
[Support]: "Send to Frigate+" button error

### ISSUE THREAD

**Issue by mdelecate** [blakeblackshear/frigate#3498]
### Describe the problem you are having

Clicking the button "Send to Frigate+" to submit an event image results in error "Setup a Frigate+ Account": 
<img width="384" alt="Screenshot 2022-07-20 at 9 27 21 PM" src="https://user-images.githubusercontent.com/12543435/179995538-b01cbf63-5aab-4a1d-9374-7c386df3600c.png">

API Key from Frigate+ account has been input, saved, and Add-On restarted: 
<img width="1044" alt="Screenshot 2022-07-20 at 9 27 52 PM" src="https://user-images.githubusercontent.com/12543435/179995619-bf95aa3c-4335-4b90-b97c-f5bb2d3b1a33.png">

I have also retried with a new key with the regenerate API Key feature on the Frigate+ website.  

### Version

0.11.0-3C46A33 /  0.11.0-beta7

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

USB

### Network connection

Wired

**Comment by mdelecate** [comment]
No problem: 

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[2022-07-20 21:39:37] frigate.app                    INFO    : Starting Frigate (0.11.0-3c46a33)
[2022-07-20 21:39:38] frigate.app                    INFO    : Creating directory: /tmp/cache
Starting migrations
[2022-07-20 21:39:38] peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
[2022-07-20 21:39:38] peewee_migrate                 INFO    : There is nothing to migrate
[2022-07-20 21:39:38] detector.coral1                INFO    : Starting detection process: 256
[2022-07-20 21:39:38] frigate.edgetpu                INFO    : Attempting to load TPU as usb:0
[2022-07-20 21:39:38] detector.coral3                INFO    : Starting detection process: 267
[2022-07-20 21:39:38] frigate.edgetpu                INFO    : Attempting to load TPU as usb:2
[2022-07-20 21:39:38] ws4py                          INFO    : Using epoll
[2022-07-20 21:39:38] detector.coral2                INFO    : Starting detection process: 260
[2022-07-20 21:39:38] frigate.edgetpu                INFO    : Attempting to load TPU as usb:1
[2022-07-20 21:39:41] frigate.edgetpu                INFO    : TPU found
[2022-07-20 21:39:41] frigate.edgetpu                INFO    : TPU found
[2022-07-20 21:39:38] frigate.app                    INFO    : Output process started: 274
[2022-07-20 21:39:38] frigate.app                    INFO    : Camera processor started for GardenFrontWindow: 292
[2022-07-20 21:39:38] frigate.app                    INFO    : Capture process started for GardenFrontWindow: 330
[2022-07-20 21:39:38] ws4py                          INFO    : Using epoll
[2022-07-20 21:39:42] frigate.edgetpu                INFO    : TPU found
[2022-07-20 21:39:50] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:59714]
[2022-07-20 21:39:51] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:59714]
[2022-07-20 21:40:28] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:42476]
[2022-07-20 21:40:34] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:42476]
[2022-07-20 21:40:55] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:59170]
[2022-07-20 21:41:01] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:59170]
[2022-07-20 21:41:49] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:45422]
[2022-07-20 21:42:16] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:45422]
[2022-07-20 21:42:25] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:33260]
[2022-07-20 21:42:45] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:33260]
```

**Comment by mdelecate** [comment]
I unfortunately don't have full SSH access right now, all I can use is the Terminal & SSH add-on. From what I understand ( as per https://developers.home-assistant.io/docs/operating-system/debugging/), I can only get unrestricted access by adding my public key with a plugged in USB drive, but I'm debugging this whole thing remotely, so that would have to wait for now.

**Comment by blakeblackshear** [comment]
I figured this out. Will be fixed in next beta.

**Comment by mdelecate** [comment]
I can confirm this is fixed in 0.11.0-rc1 🙌
